spec/registry.yaml

```diff
@@ -0,0 +1,89 @@
+groups:
+  - id: registry.graphql
+    type: attribute_group
+    display_name: GraphQL Attributes
+    brief: "This document defines attributes for GraphQL operations and resolvers."
+    attributes:
+      - id: graphql.operation.name
+        brief: "The name of the operation being executed."
+        type: string
+        stability: development
+        examples: ["FindBookById", "GetUserProfile"]
+        note: >
+          This represents the operation name as specified in the GraphQL query document.
+          When the operation name is not provided, this attribute SHOULD be omitted.
+      - id: graphql.operation.type
+        brief: "The type of the operation being executed."
+        stability: development
+        type:
+          members:
+            - id: query
+              value: "query"
+              brief: "GraphQL query operation"
+              stability: development
+            - id: mutation
+              value: "mutation"
+              brief: "GraphQL mutation operation"
+              stability: development
+            - id: subscription
+              value: "subscription"
+              brief: "GraphQL subscription operation"
+              stability: development
+        examples: ["query", "mutation", "subscription"]
+      - id: graphql.operation.hash
+        brief: "The hash of the operation document."
+        type: string
+        stability: development
+        examples:
+          [
+            "sha256:400483f38c08e8a3d3b972409c9dfb8e4a326e1b1940864932acd9f873d8664c",
+          ]
+        note: >
+          The hash algorithm used SHOULD be specified as part of the value (e.g., "sha256:...").
+          This can be used for monitoring operation distribution and caching strategies.
+      - id: graphql.operation.id
+        brief: >
+          The document identifier for persisted operations.
+        type: string
+        stability: development
+        examples: ["aa3e37c1bf54708e93f12c137afba004"]
+        note: >
+          This is a hash or identifier of the document provided by the user to identify
+          persisted operations. This attribute SHOULD only be set when using persisted operations.
+      - id: graphql.selection.name
+        brief: "The name of the selection that is being resolved. Either the field name or an alias."
+        type: string
+        stability: development
+        examples: ["newAddress", "bookTitle"]
+        note: >
+          If the field has an alias, this SHOULD be the alias name. Otherwise, it SHOULD be the field name.
+      - id: graphql.selection.path
+        brief: "The path of the selection that is being resolved."
+        type: string
+        stability: development
+        examples: ["/persons/0/address", "person[0].address"]
```

Comment:
We have to settle here on 1 format

Reply:
Given the only mention of ["path" in the spec](https://github.com/graphql/graphql-spec/blame/51337a9b820e296fa7d03ae77d534cb4b247c201/spec/Section%207%20--%20Response.md#L109-L114) defines it as a heterogenous array of strings and integers, and that [JSON](https://spec.graphql.org/October2021/#sec-JSON-Serialization) is documented as the most common serialization format, I'm inclined to prefer the [bracket notation](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Property_accessors#bracket_notation) option.